mm2src/mm2_tests.rs

```diff
@@ -8320,3 +8320,98 @@ fn test_get_orderbook_with_same_orderbook_ticker() {
+fn test_get_raw_transaction() {
+    let bob_passphrase = get_passphrase(&".env.client", "BOB_PASSPHRASE").unwrap();
+    let bob_coins_config = json!([
+        {"coin":"RICK","asset":"RICK","rpcport":8923,"txversion":4,"overwintered":1,"protocol":{"type":"UTXO"}},
+        {"coin":"ETH","name":"ethereum","protocol":{"type":"ETH"},"rpcport":80},
+    ]);
+
+    let mm_bob = MarketMakerIt::start(
+        json! ({
+            "gui": "nogui",
+            "netid": 9998,
+            "myipaddr": env::var ("BOB_TRADE_IP") .ok(),
+            "rpcip": env::var ("BOB_TRADE_IP") .ok(),
+            "canbind": env::var ("BOB_TRADE_PORT") .ok().map (|s| s.parse::<i64>().unwrap()),
+            "passphrase": bob_passphrase,
+            "coins": bob_coins_config,
+            "rpc_password": "pass",
+            "i_am_seed": true,
+        }),
+        "pass".into(),
+        local_start!("bob"),
+    )
+    .unwrap();
+    let (_bob_dump_log, _bob_dump_dashboard) = mm_bob.mm_dump();
+    log!({"Bob log path: {}", mm_bob.log_path.display()});
+
+    let electrum = block_on(mm_bob.rpc(json!({
+        "userpass": "pass",
+        "method": "electrum",
+        "coin": "RICK",
+        "servers": [{"url":"electrum1.cipig.net:10017"},{"url":"electrum2.cipig.net:10017"},{"url":"electrum3.cipig.net:10017"}],
+        "mm2": 1,
+    }))).unwrap();
+    assert_eq!(
+        electrum.0,
+        StatusCode::OK,
+        "RPC «electrum» failed with {} {}",
+        electrum.0,
+        electrum.1
+    );
+    log!({ "enable RICK: {:?}", electrum });
+    let enable_rick_res: Json = json::from_str(&electrum.1).unwrap();
+    let _rick_address = enable_rick_res["address"].as_str().unwrap();
+
+    let (_dump_log, _dump_dashboard) = mm_bob.mm_dump();
+    log!({ "log path: {}", mm_bob.log_path.display() });
+    block_on(mm_bob.rpc(json! ([{
+        "userpass": mm_bob.userpass,
+        "method": "enable",
+        "coin": "ETH",
+        "urls": &["https://ropsten.infura.io/v3/c01c1b4cf66642528547624e1d6d9d6b"],
+        // Dev chain swap contract address
+        "swap_contract_address": "0xa09ad3cd7e96586ebd05a2607ee56b56fb2db8fd",
+        "mm2": 1,
+    }
+    ])))
+    .unwrap();
+
+    let eth_raw_tx = block_on(mm_bob.rpc(json!({
+      "userpass": mm_bob.userpass,
+      "mmrpc":"2.0",
+      "method":"get_raw_transaction",
+      "params":{
+        "coin":"ETH",
+        "tx_hash": "0xbdef3970c00752b0dc811cd93faadfd75a7a52e6b8e0b608c5519edcad801359"
+      },
+      "id": 0
+    })))
+    .unwrap();
+
+    let rick_raw_tx = block_on(mm_bob.rpc(json!({
+      "userpass":mm_bob.userpass,
+      "mmrpc":"2.0",
+      "method":"get_raw_transaction",
+      "params":{
+        "coin":"RICK",
+        "tx_hash": "989360b0225b4e05fa13643e2e306c8eb5c52fa611615dfd30195089010b1c7b"
+      },
+      "id": 0
+    })))
+    .unwrap();
+
+    let eth_response = serde_json::from_str::<RpcSuccessResponse<RawTransactionResponse>>(&eth_raw_tx.1).unwrap();
```

Comment:
We use the shorter `json` alias for `serde_json`. Also, `let eth_response: <RpcSuccessResponse<RawTransactionResponse>> = ...` will be a bit easier to read.